wamr-compiler/main.c

```diff
@@ -57,12 +57,13 @@ print_help()
     printf("  --disable-simd            Disable the post-MVP 128-bit SIMD feature:\n");
     printf("                              currently 128-bit SIMD is supported for x86-64 and aarch64 targets,\n");
     printf("                              and by default it is enabled in them and disabled in other targets\n");
-    printf("  --disable-ref-types       Disable the MVP reference types feature\n");
+    printf("  --disable-ref-types       Disable the MVP reference types feature, it will be disabled forcibly if GC enabled\n");
```

Comment:
```C
printf(" --disable-ref-types Disable the MVP reference types feature, it will be disabled forcibly if\n");
printf("                     GC is enabled\n");